.github/workflows/publish-to-pypi.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       run: python -m pip install setuptools wheel
 
     - name: Fix up version string
+      if: !startsWith(github.ref, 'refs/tags')
```

Comment:
Can you also move the comment "The step is only necessary for testing purpose" here, so we can better why we use `!startsWith(github.ref, 'refs/tags')`.

Reply:
Ok, done at 82290d66716a097d12c135ad713f2f8f3f8057c9